src/simba/dashboard/zonal_manager/revenue_branch_performance_card.html

```
@@ -0,0 +1,182 @@
+<div id="hs-pro-tabs-sg-open" role="tabpanel" aria-labelledby="hs-pro-tabs-sg-item-open"
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)